scripts/tools/python3.sh

```diff
@@ -0,0 +1,26 @@
+# shellcheck shell=bash
+# shellcheck disable=SC2034
+
+TOOLS+=(python3)
+
+PYTHON3_VERSION=3.3
+
+function python3_version {
+    # Only return a version when virtualenv is setup and activated.
```

Comment:
But it's us who activate the vitrualenv (below) no? Which case are we trying to avoid here?

Reply:
I can no longer remember; I suspect it is no longer serving any valid purpose. I've removed it in ce33a56. We can always add it back later if we find some breakage. But in my simple tests using `yamllint` it didn't seem to be useful.